### ISSUE TITLE
Start kernel in single threaded mode

### ISSUE THREAD

**Issue by r00tus3r** [lkl/linux#498]
Hi, I'm trying to find a way to start the kernel in single thread mode, but haven't been successful. Is it possible to do that?

I found the `singlecpu` option in here: https://github.com/lkl/linux/blob/master/Documentation/lkl.txt#L226

But I guess that is just single cpu pinning and not single threaded mode?

Also, I asked this question in lkl slack but didn't get a response.

**Comment by ddiss** [comment]
> But I guess that is just single cpu pinning and not single threaded mode?

Yes, lkl generally spawns one host thread for each kernel thread - see `tools/lkl/lib/posix-host.c`.
